src/openforms/registrations/contrib/zgw_apis/tests/test_backend.py

```diff
@@ -304,6 +328,351 @@ def test_submission_with_zgw_backend(self, m):
+            create_eio_body["informatieobjecttype"],
+            "https://catalogi.nl/api/v1/informatieobjecttypen/1",
+        )
+
+        create_zio = m.request_history[4]
+        create_zio_body = create_zio.json()
+        self.assertEqual(create_zio.method, "POST")
+        self.assertEqual(
+            create_zio.url, "https://zaken.nl/api/v1/zaakinformatieobjecten"
+        )
+        self.assertEqual(create_zio_body["zaak"], "https://zaken.nl/api/v1/zaken/1")
+        self.assertEqual(
+            create_zio_body["informatieobject"],
+            "https://documenten.nl/api/v1/enkelvoudiginformatieobjecten/1",
+        )
+
+        create_rol = m.request_history[7]
+        create_rol_body = create_rol.json()
+        self.assertEqual(create_rol.method, "POST")
+        self.assertEqual(create_rol.url, "https://zaken.nl/api/v1/rollen")
+        self.assertEqual(create_zio_body["zaak"], "https://zaken.nl/api/v1/zaken/1")
+        self.assertEqual(
+            create_rol_body["roltype"],
+            "https://catalogus.nl/api/v1/roltypen/1",
+        )
+        self.assertEqual(
+            create_rol_body["betrokkeneIdentificatie"],
+            {
+                "handelsnaam": "ACME",
+                "vestigingsNummer": "87654321",
+                "verblijfsadres": {"postcode": "1000 AA"},
+            },
+        )
+
+        create_status = m.request_history[9]
+        create_status_body = create_status.json()
+        self.assertEqual(create_status.method, "POST")
+        self.assertEqual(create_status.url, "https://zaken.nl/api/v1/statussen")
+        self.assertEqual(create_status_body["zaak"], "https://zaken.nl/api/v1/zaken/1")
+        self.assertEqual(
+            create_status_body["statustype"],
+            "https://catalogus.nl/api/v1/statustypen/1",
+        )
+
+        create_attachment = m.request_history[10]
+        create_attachment_body = create_attachment.json()
+        self.assertEqual(create_attachment.method, "POST")
+        self.assertEqual(
+            create_attachment.url,
+            "https://documenten.nl/api/v1/enkelvoudiginformatieobjecten",
+        )
+        self.assertEqual(create_attachment_body["bestandsnaam"], attachment.file_name)
+        self.assertEqual(create_attachment_body["formaat"], attachment.content_type)
+
+        relate_attachment = m.request_history[11]
+        relate_attachment_body = relate_attachment.json()
+        self.assertEqual(relate_attachment.method, "POST")
+        self.assertEqual(
+            relate_attachment.url, "https://zaken.nl/api/v1/zaakinformatieobjecten"
+        )
+        self.assertEqual(
+            relate_attachment_body["zaak"], "https://zaken.nl/api/v1/zaken/1"
+        )
+        self.assertEqual(
+            relate_attachment_body["informatieobject"],
+            "https://documenten.nl/api/v1/enkelvoudiginformatieobjecten/2",
+        )
+
+    def test_submission_with_zgw_backend_with_vestiging_initiator_kvk_only(self, m):
```

Comment:
I'm missing a test case where an optional `vestigingsNummer` field is present with an empty string as value.